packages/fuel-indexer-lib/src/utils.rs

```diff
@@ -203,3 +208,140 @@ pub mod index_utils {
         s
     }
 }
+
+pub fn format_exec_msg(exec_name: &str, path: Option<String>) -> String {
+    if let Some(path) = path {
+        rightpad_whitespace(&path, defaults::MESSAGE_PADDING)
+    } else {
+        rightpad_whitespace(
+            &format!("Can't locate {exec_name}."),
+            defaults::MESSAGE_PADDING,
+        )
+    }
+}
+
+pub fn find_executable_with_msg(exec_name: &str) -> (String, Option<String>, String) {
+    let (emoji, path) = find_executable(exec_name);
+    let p = path.clone();
+    (emoji, path, format_exec_msg(exec_name, p))
+}
+
+pub fn find_executable(exec_name: &str) -> (String, Option<String>) {
+    match Command::new("which").arg(exec_name).output() {
+        Ok(o) => {
+            let path = String::from_utf8_lossy(&o.stdout)
+                .strip_suffix('\n')
+                .map(|x| x.to_string())
+                .unwrap_or_else(String::new);
+
+            if !path.is_empty() {
+                (
+                    center_align("✅", defaults::SUCCESS_EMOJI_PADDING),
+                    Some(path),
+                )
+            } else {
+                (center_align("⛔️", defaults::FAIL_EMOJI_PADDING - 2), None)
+            }
+        }
+        Err(_e) => (center_align("⛔️", defaults::FAIL_EMOJI_PADDING), None),
+    }
+}
+
+pub fn center_align(s: &str, n: usize) -> String {
+    format!("{s: ^n$}")
+}
+
+pub fn rightpad_whitespace(s: &str, n: usize) -> String {
+    format!("{s:0n$}")
+}
+
+
+// NOTE: We aren't using this now but will leave it here, as I think we'll revisit
+pub fn create_forc_postgres_database(config: &IndexerConfig) -> Result<()> {
```

Comment:
As mentioned in the PR description, this is ultimately where we wanna go - automatically creating the database. However, there's a lot of effort involved in this - so making the tradeoff to just merge without this for now, then follow up in a separate issue.